server/block/sign.go

```diff
@@ -178,7 +177,10 @@ func (s Sign) NeighbourUpdateTick(pos, _ cube.Pos, w *world.World) {
 
 // EncodeBlock ...
 func (s Sign) EncodeBlock() (name string, properties map[string]any) {
-	woodType := strings.Replace(s.Wood.String(), "_", "", 1) + "_"
+	woodType := s.Wood.String() + "_"
+	if woodType == "dark_oak_" {
+		woodType = "darkoak_"
+	}
```

Comment:
Can you compare s.Wood to DarkOakWood() instead, i'd also change the statement below to compare to OakWood()